README.md

```diff
@@ -85,9 +85,10 @@ Using the script [tests/automated_install.sh](./tests/automated_install.sh) on M
 will require [`brew`](http://brew.sh) to be available/installed.
 
 Note: To run more than one test cluster at a time with VirtualBox, one may
-      set in `tests/automated_install.sh` their desired cluster name prefix.
+      export BACH_CLUSTER_PREFIX to set their desired cluster name prefix.
       This will namespace the cluster's virtual machines to not collide on the
-      hypervisor. One also needs to ensure their management, float and storage
+      hypervisor.  Lacking that tests/automated_install.sh will use a default 
```

Comment:
Sentence fragment `Lacking that tests/automated_install.sh will use a default`?

Reply:
Fixed wording